packages/test/src/mock-native-worker.ts

```diff
@@ -163,11 +159,12 @@ export class Worker extends RealWorker {
   }
 
   public constructor(workflowCreator: WorkflowCreator, opts: CompiledWorkerOptions) {
-    // Worker.create() accesses Runtime.instance(), which has some side effects that would not happen (or that would
-    // happen too late) when creating a MockWorker. Force the singleton to be created now, if it doesn't already exist.
-    Runtime.instance();
+    const logger = withMetadata(Runtime.instance().logger, {
+      logSource: LogSource.worker,
+      taskQueue: opts.taskQueue ?? 'default',
```

Comment:
When would this be empty? Replay worker? Either document why this is here or just omit the task queue completely.

Reply:
Can't happen in this palce. Removed.